pages/content/posts/swag-at-tpac-anaheim.md

```diff
@@ -0,0 +1,38 @@
+---
+author: "Will Bamberg"
+title: "Talking about security docs at TPAC Anaheim"
+date: "2024-10-18"
+tags: ["announcements"]
+ShowToc: false
+ShowBreadCrumbs: false
+excerpt: "At TPAC Anaheim, we talked with security experts about security documentation needs."
```

Comment:
```suggestion
title: "Talking about security docs at TPAC 2024"
date: "2024-10-18"
tags: ["announcements"]
ShowToc: false
ShowBreadCrumbs: false
excerpt: "At TPAC Anaheim, the Open Web Docs team spoke with security experts about security documentation needs."
```
Thinking: put Anaheim in one location and Anaheim in the other.
I suggested the updates to the excerpt so a bit more detail ends up on Google and our blog landing page.